app/Main.hs

```diff
@@ -22,16 +23,18 @@ main = do
     let configFile = case args of
             [cnf] -> cnf
             _ -> error usage
-    run configFile repostUpdates
+    run configFile
   where
```

Comment:
Not sure that I like this change, but I was lazy to add many instances for `RssT`, `DiscourseT` and `GitterT`.